Taarafo.Core/Brokers/Storages/StorageBroker.PostImpressions.cs

```diff
@@ -39,5 +40,18 @@ public async ValueTask<PostImpression> UpdatePostImpressionAsync(PostImpression
 
             return postImpressionEntityEntry.Entity;
         }
+
+        public async ValueTask<PostImpression> SelectPostImpressionByIdsAsync(
+            Guid PostId,
+            Guid ProfileId)
+        {
+            using var broker =
+                new StorageBroker(this.configuration);
+
+            broker.ChangeTracker.QueryTrackingBehavior =
+                QueryTrackingBehavior.NoTracking;
+
+            return await broker.PostImpressions.FindAsync(PostId, ProfileId);
```

Comment:
Does EF know that this method needs to be queried based on these two `Ids`?

Reply:
I don't think the EF knows that the method needs to be queried based on the two Ids.  How would I be able to verify this?

Reply:
> Does EF know that this method needs to be queried based on these two `Ids`?

@ElbekDeveloper / @evangelinedrink, yes EF can find 

> I don't think the EF knows that the method needs to be queried based on the two Ids. How would I be able to verify this?

@evangelinedrink / @ElbekDeveloper EF can do a find for a composite key.  To give EF that info, we would need to add that fluent declaration to the `StorageBroker.PostImpressions.Configurations.cs` file as has been done already.

```
            modelBuilder.Entity<PostImpression>()
                   .HasKey(postImpression =>
                       new { postImpression.PostId, postImpression.ProfileId });
```
See this for more info on composite keys: [https://www.learnentityframeworkcore.com/configuration/fluent-api/haskey-method](https://www.learnentityframeworkcore.com/configuration/fluent-api/haskey-method)